[2021-11-01 03:40:27,381] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.showresults manual__2021-11-01T03:40:21.717196+00:00 [queued]>
[2021-11-01 03:40:27,393] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.showresults manual__2021-11-01T03:40:21.717196+00:00 [queued]>
[2021-11-01 03:40:27,393] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 03:40:27,393] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-01 03:40:27,393] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 03:40:27,408] {taskinstance.py:1262} INFO - Executing <Task(PostgresOperator): showresults> on 2021-11-01 03:40:21.717196+00:00
[2021-11-01 03:40:27,412] {standard_task_runner.py:52} INFO - Started process 601 to run task
[2021-11-01 03:40:27,415] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'showresults', 'manual__2021-11-01T03:40:21.717196+00:00', '--job-id', '148', '--raw', '--subdir', 'DAGS_FOLDER/checkModel.py', '--cfg-path', '/tmp/tmpn0z8qevk', '--error-file', '/tmp/tmphcpwgtgo']
[2021-11-01 03:40:27,415] {standard_task_runner.py:77} INFO - Job 148: Subtask showresults
[2021-11-01 03:40:27,448] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.showresults manual__2021-11-01T03:40:21.717196+00:00 [running]> on host e4a42aadf2ec
[2021-11-01 03:40:27,489] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=showresults
AIRFLOW_CTX_EXECUTION_DATE=2021-11-01T03:40:21.717196+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-01T03:40:21.717196+00:00
[2021-11-01 03:40:27,496] {base.py:79} INFO - Using connection to: id: postgres_default. Host: 172.23.0.3, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2021-11-01 03:40:27,499] {dbapi.py:225} INFO - Running statement: select * from results;, parameters: None
[2021-11-01 03:40:27,500] {dbapi.py:233} INFO - Rows affected: 418
[2021-11-01 03:40:27,510] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=showresults, execution_date=20211101T034021, start_date=20211101T034027, end_date=20211101T034027
[2021-11-01 03:40:27,546] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-01 03:40:27,570] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
